File: week2/task4/task4.js
function getNumber(index) {
  let number = 0;
  for (let i = 1; i <= index; i++) {
    if (i % 3 == 0) {
      number--;
    }
    else {
      number += 4;
    }
  }
  return number;
}
getNumber(1); // print 4
getNumber(5); // print 15
getNumber(10); // print 25
getNumber(30); // print 70

File: week2/task4/task4.py
def get_number(index):
    number = 0
    for i in range(1, index + 1):
        if i % 3 == 0:
            number -= 1
        else:
            number += 4
    print(number)
    # return number


get_number(1)  # print 4
get_number(5)  # print 15
get_number(10)  # print 25
get_number(30)  # print 70
